api/routes/fastencipher2.js

```diff
@@ -30,39 +30,85 @@ const sc = require('../lib/SystemCheck.js');
  * -- NO AUTHENTICATION IS NEEDED FOR URL /message
  * -- message is encrypted already
  */
-router.post('/message/:gid',
-    sc.isInitialized,
-    encryption.decrypt,
-    sc.debugInfo,
 
-    (req, res, next) => {
-      const gid = req.params.gid;
+const msgHandler = (req, res, next) => {
+  const gid = req.params.gid;
+  const streaming = (req.body.message && req.body.message.obj && req.body.message.obj.streaming) || false;
+  (async () => {
+    if (streaming) {
```

Comment:
use a separate function to handle GET+event, do not change the existing POST function.

keep logic separated.